lib/siftsciex/decision.ex

```diff
@@ -54,4 +99,12 @@ defmodule Siftsciex.Decision do
 
     time
   end
+
+  defp request_url(entity_id, entity_type, account_id) do
+    "#{base_url()}/#{account_id}/#{entity_type}s/#{entity_id}/decisions"
+  end
```

Comment:
This is the request for the [Decision Status API](https://sift.com/developers/docs/curl/decisions-api/decision-status), right?

If so, there are more URLs to cover if we want to support `sessions` and `content`:

```
/v3/accounts/{accountId}/{users}/{userId}/decisions
/v3/accounts/{accountId}/orders/{orderId}/decisions
/v3/accounts/{accountId}/users/{userId}/sessions/{sessionId}/decisions
/v3/accounts/{accountId}/users/{userId}/content/{contentId}/decisions
```

Reply:
Hmmm I did not notice that! Thank you. Man, why can't they just be consistent in their routes? :joy: